start.sh

```diff
@@ -24,6 +24,11 @@ set -x
 
 python manage.py migrate
 
+if [ "${CACHE_BACKEND:-""}" == "django.core.cache.backends.db.DatabaseCache" ]; then
+    echo "Database cache set, cerating cache table"
+    python manage.py createcachetable
```

Comment:
Change to "Database cache set; creating cache table"?

Reply:
Thanks, no spell checker in the start script.